### ISSUE TITLE
Exit stealth mode and add `/about/foundation` page to `materialsproject.org`

### ISSUE THREAD

**Issue by janosh** [materialsproject/foundation#16]
### Problem

As discussed in this morning's foundation meeting, we think the foundation has become established enough to exit stealth mode and become more visible, partially with the intent of attracting a more diverse set of future members from the MP community.

### Proposed Solution

Create a dedicated page at `materialsproject.com/about/foundation` introducing visitors to its objectives, current members and how to get involved.

I opened a PR with a proposal for what such a page might look like. Feedback welcome!

https://github.com/materialsproject/web/pull/895

![Screenshot 2023-11-06 at 11 11 10 AM](https://github.com/materialsproject/foundation/assets/30958850/084d39d5-c1a2-4a62-832c-af1d7954a7b3)

**Comment by JaGeo** [comment]
Could we add text about how to potentially get involved? I know this might also need a decision from our side first. This might also help with the diversity point that @mkhorton raised.
I think at the moment it is not very clear to anyone outside of MP (maybe even partially inside) how one would become a part of the foundation.

**Comment by janosh** [comment]
Yes, that's perhaps the most important content. This issue is to solicit input from everyone on how we want to phrase that.

**Comment by yang-ruoxi** [comment]
@janosh I feel very disrespected that I am not included in the member list as a core infra member of MP, which is ironic considering this is raised along the topic of diversity. Could you clarify your intent behind this? Thank you.

**Comment by janosh** [comment]
No disrespect intended. My understanding is MP staff member and foundation member are two separate things, one doesn't imply the other. I wasn't aware you are or want to be part of the foundation. This PR is a draft and just includes everyone who was in today's Zoom meeting to get started.

**Comment by shyuep** [comment]
@yang-ruoxi Just to be clear - Foundation and MP staff *are* two separate things. Foundation members and software maintainers are also two separate things. For example, I am lead maintainer for pymatgen but @janosh did not include me in the list of foundation members. There is no reason to feel "disrespected". If you want to be part of the foundation, you just have to ask. But if you are part of the foundation, be prepared to participate and contribute. So be clear on whether you want in or out.

**Comment by yang-ruoxi** [comment]
@shyuep Thank you for your input. For context I have been participating in all past foundation meetings except for today (as I didn't feel well), I consider that as "I am part of it", hence the sense of being ignored. But do let me know if this is not obvious enough and I can further clarify. Thanks!

**Comment by shyuep** [comment]
@yang-ruoxi In that case, by all means add yourself to the page... :-) Good luck. I hope you know what you signed up for.

**Comment by JaGeo** [comment]
In any case, we should, of course, include everyone that is involved in the foundation. I think a few other people from inside MP are missing as well (maybe, the mailing list that Alex used for the invitation helps)

**Comment by utf** [comment]
I'd like to echo Ruoxi's point. She has been involved in almost every meeting so far and I would very much consider her a member of the Foundation.

I'm sure her omission was a simple mistake with no bad intentions. That said, we want the foundation to be a welcoming place for all members and, personally, I found the subsequent discussion questioning her involvement and commitment inappropriate.

**Comment by shyuep** [comment]
@utf I don't recall anyone questioning her involvement or commitment. All anyone did was to clarify that being a foundation member is a **choice** and separate from whether you are an MP staff member. I guess it is now fashionable to just call out simple clarifications for being "inappropriate"?

**Comment by janosh** [comment]
Just to further clarify, there was no ill will on my part. I felt the urge to get this PR opened before my next meeting yesterday to get the ball rolling and start gathering feedback. I now regret rushing it and not being more deliberate about the member list.

Following @JaGeo's advice I looked at the email list for the meeting reminder @utf sent out:

Alex Ganose
Anubhav Jain
Andrew Rosen
EWCSpotteSmith@lbl.gov
Janine George
Jason Munro
Kristin Persson
Karlo Berket
Ryan Kingsbury
Matthew Horton
Shyue Ping Ong
Patrick Huck
ruoxiyang@lbl.gov
scholia@lbl.gov
Jimmy-Xuan Shen

Looks like we may also be missing @espottesmith, scholia@lbl.gov, Karlo Berket, Anubhav, Kristin and Shyue. I'll leave it to others to clarify who is involved to what extent.

@yang-ruoxi In any case, I feel like your concern would have been better served by an email rather than a heated discussion in public.

**Comment by yang-ruoxi** [comment]
@janosh Let me get this clear: I understand this PR is a draft and I am sure the omission was a simple mistake with no bad intention. What frustrated me is when I raised the question, instead of simply being added, I wasn’t acknowledged well, and that I had to “prove” I am part of it, and to be educated the difference between the foundation and the staff, something I was well aware of.  I feel I had to go through this while other members don’t.  

Ironically, this exact issue was created as an effort to address diversity and inclusivity. 

So back to the topic of this thread, maybe we could reflect more on how we can be more inclusive towards collaborators? I doubt creating another web page that lists all current members would do it; after all, seeing the skewed gender ratio there may not be that helpful, not to mention there already exists such a section on web from a year ago. Maybe we could stress more on the code of conduct, our values and our principles instead. 

I initially asked the question publicly because I feel inclusion is important and should be discussed publicly. However as the discussion drags on I find it unproductive and emotionally taxing, so I will stop responding from here.

**Comment by mkhorton** [comment]
Thank you @janosh for opening this thread with the aim of improving visibility of the Foundation, and to @yang-ruoxi for the specific suggestions of stressing code of conduct, values and principles.

I mentioned diversity and representation during the last Foundation meeting because, factually, we have very poor diversity at present. I would like to make progress here.

I think it is ok to have these discussions--even uncomfortable discussions--in public. It is an opportunity to show how we, as a group, respond to feedback. Open source development happens in public too. If we want to talk about ideals like "being welcoming", "being inclusive", we need to follow through and ask what this looks like in practice too.

I want to emphasize that these are my personal views. I will naturally defer to MP staff and those funded by MP on how best to run the Foundation. I wanted to share these views as a participant in the open source community. These views are also maybe self-evident or already known to those reading this, but I think can be helpful to state nevertheless.

First, we can recognize that we make mistakes, and can have misunderstandings. This is ok and normal. I don't think any less of a colleague who makes a mistake. I have enjoyed working with everyone in this thread. Misunderstandings are common too, especially over text. However, when these events do occur, it is important not to gloss over them.

I don't want to re-open discussion of this particular example myself, since I do not feel like that's appropriate for me to do, but perhaps we can talk more broadly about what happens if someone feels disrespected, and then communicates that with the group. There is the feeling of being disrespected itself, and also the facts and context that caused that feeling. Both need to be addressed, because the bare facts in one incident might hide a pattern of behavior or a larger problem that has not been given appropriate attention.

I have done plenty of DEI trainings, but frankly I am still learning, and still have a way to go myself. I am not the right person to be educating anyone here. But I can still share some things that I've found important, and these include learning about (1) implicit/unconscious bias, and especially "presumed incompetence", which many people from underrepresented groups face, (2) an issue with lack of recognition, whereby two people might equally contribute to a project/meeting/etc., but one of those two people might have their contributions better remembered or better recognized, and (3) the danger of personal admonishments when these come from someone in a culturally dominant position, since these can carry a lot of extra baggage and weight, that would lead to an unwelcome atmosphere.

So what can we do? When things like this happen we can listen. If we feel disrespected ourselves, we can speak up and give others an opportunity to correct. If we make a mistake or simply mis-speak and cause offense ourselves, we can apologise. We can question our own actions, and try to notice better where we might have shown implicit bias (even if not consciously aware at the time!) And we can give each other grace, because I genuinely believe we all want the best for each other, and I have a lot of respect for everyone here.

Apologies for the long message but it felt important. I am really glad to be part of this community. I'd like to stay part of it. But I can only participate knowing that everyone's voices are respected.
